transcom/mymove
# B 18429 ssw model to service object conversion

## [B-18429 PPM SSW Model to Service Object Conversion](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a883383)

Integration PR: [B-18429-INT](https://github.com/transcom/mymove/pull/11735)

## Summary

The SSW was converted from a model to a service object, and as much data as possible was changed from personallyprocuredmove to PPMShipment. There are still obligations and signed certifications that will require downstream work, but this restructure was more time sensitive due to downstream dependencies.

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Pull the branch, ensure the SSW files are correctly structured and no personallyprocuredmove code is still active.
2. Run the SSW Testing suite, ensure all tests pass and all tests have been reviewed.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
